### DISCUSSION TITLE
Update Surricata rules

### DISCUSSION THREAD

**Discussion by cbeek-r7** [telekom-security/tpotce#1223] [Q&A] [answered]
How can I update the Surricata rules to use for example the Pro Ruleset?

**Comment by t3chn0m4g3** [comment, marked as answer]
Great question, I thought we already documented this 😄 

Stop T-Pot and go to `/opt/tpot/etc/tpot.yml` and you will find this [section](https://github.com/telekom-security/tpotce/blob/2641d1e74306fafa9c43f08fb6927e3d9781c936/etc/compose/standard.yml#L492-L508). Uncomment the lines and adjust accordingly and then start T-Pot again.

**Reply by cbeek-r7** [reply]
Fantastic! Thank you!
